File: Python/Fizz Buzz.py
class Solution(object):
    def fizzBuzz(self, n):
        """
        :type n: int
        :rtype: List[str]
        """
        return [str(i) if (i % 3 != 0 and i % 5 != 0) else 'FizzBuzz' if (i % 15 == 0) else ('Fizz' if i % 3 == 0 else 'Buzz') for i in xrange(1, n + 1)]

File: cpp/Fizz Buzz.cpp
#include <cstdio>
class Solution {
public:
    vector<string> fizzBuzz(int n) {
        vector<string> result;
        for (int i = 1; i <= n; ++i) {
            if (i % 3 == 0 && i % 5 == 0) {
                result.push_back(string("FizzBuzz"));
            } else if (i % 3 == 0) {
                result.push_back(string("Fizz"));
            } else if (i % 5 == 0) {
                result.push_back(string("Buzz"));
            } else {
                result.push_back(to_string(i));
            }
        }
        return result;
    }
};
